packages/mdc-grid-list/mdc-grid-list.scss

```diff
@@ -0,0 +1,237 @@
+@import "@material/theme/mixins";
+
+$mdc-grid-list-tile-width: 200px;
+$mdc-grid-list-tile-secondary-padding: 16px;
+$mdc-grid-list-tile-secondary-padding-narrow: 8px;
+$mdc-grid-list-oneline-cap-secondary-height: 48px;
+$mdc-grid-list-twoline-cap-secondary-height: 68px;
+$mdc-grid-list-tile-secondary-icon-size: 24px;
+
+// postcss-bem-linter: define grid-list
+// stylelint-disable plugin/selector-bem-pattern
+@mixin mdc-grid-list--tile-aspect($width-height-ratio) {
+  .mdc-grid-tile .mdc-grid-tile__primary {
+    padding-bottom: calc(100% / #{$width-height-ratio});
+  }
+}
+
+@mixin mdc-grid-list--tile-gutter($gutter-width) {
+  .mdc-grid-tile {
+    margin: $gutter-width/2 0;
+    padding: 0 $gutter-width/2;
+
+    &__secondary {
+      left: $gutter-width/2;
+      width: calc(100% - #{$gutter-width});
+    }
+  }
+
+  .mdc-grid-list__tiles {
+    margin: $gutter-width/2 auto;
+  }
+}
+// stylelint-enable plugin/selector-bem-pattern
+
+.mdc-grid-list {
+  /*
+   *  Default grid list style:
+   *  - mdc-grid-list--tile-gutter-4
+   *  - mdc-grid-list--tile-aspect-1x1
+   *  - mdc-grid-list--oneline-caption
+   *  - mdc-grid-list--footer-caption
+   *  - mdc-grid-list--no-icon
+   *
+   */
+  @include mdc-grid-list--tile-aspect(1);
+  @include mdc-grid-list--tile-gutter(4px);
+
+  // stylelint-disable plugin/selector-bem-pattern
+  .mdc-grid-tile {
+    &__secondary {
+      bottom: 0;
+      height: $mdc-grid-list-oneline-cap-secondary-height;
+      padding: $mdc-grid-list-tile-secondary-padding;
+    }
+
+    &__icon {
+      font-size: 0;
+    }
+  }
+  // stylelint-enable plugin/selector-bem-pattern
+
+  &__tiles {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    margin: 0;
+    padding: 0;
+  }
+}
+
+// stylelint-disable plugin/selector-bem-pattern
+.mdc-grid-list--tile-gutter-1 {
+  @include mdc-grid-list--tile-gutter(1px);
+}
+
+.mdc-grid-list--tile-aspect-16x9 {
+  @include mdc-grid-list--tile-aspect(16 / 9);
+}
+
+.mdc-grid-list--tile-aspect-3x2 {
+  @include mdc-grid-list--tile-aspect(3 / 2);
+}
+
+.mdc-grid-list--tile-aspect-2x3 {
+  @include mdc-grid-list--tile-aspect(2 / 3);
+}
+
+.mdc-grid-list--tile-aspect-4x3 {
+  @include mdc-grid-list--tile-aspect(4 / 3);
+}
+
+.mdc-grid-list--tile-aspect-3x4 {
+  @include mdc-grid-list--tile-aspect(3 / 4);
+}
+
+.mdc-grid-list--twoline-caption {
+  .mdc-grid-tile__secondary {
+    height: $mdc-grid-list-twoline-cap-secondary-height;
+  }
+}
+
+.mdc-grid-list--header-caption {
+  .mdc-grid-tile__secondary {
+    top: 0;
+    bottom: auto;
+  }
+}
+
+.mdc-grid-list--with-icon-align-start {
+  .mdc-grid-tile__secondary {
+    padding-right: $mdc-grid-list-tile-secondary-padding-narrow;
+    padding-left: $mdc-grid-list-tile-secondary-padding * 2 + $mdc-grid-list-tile-secondary-icon-size;
+
+    .mdc-grid-tile__icon {
+      left: $mdc-grid-list-tile-secondary-padding;
+      font-size: $mdc-grid-list-tile-secondary-icon-size;
+    }
+  }
+
+  @include mdc-rtl {
```

Comment:
I believe this, and below, should be

`@include mdc-rtl(".mdc-grid-list", true)`

to check, try putting `dir="rtl"` _directly_ on the grid list and see if the changes apply